testdata/.gitignore

```diff
@@ -8,3 +8,4 @@
 /psiphon_config.json
 /psiphon_unit_tests/
 /test-download-resources-idempotent*/
+/fsx/
```

Comment:
This directory should not be ignored. On the contrary, we want to create files and directories inside of `./internal/fsx/testdata` so that we don't need to create them during tests (see my above comments).